docs/api/apiv3/components/schemas/file_link_model.yml

```diff
@@ -114,13 +137,163 @@ example:
     lastModifiedAt: '2021-12-20T14:00:13.987Z'
     createdByName: Luke Skywalker
     lastModifiedByName: Anakin Skywalker
+  _embedded:
+    storage:
+      id: 1337
+      _type: Storage
+      href: https://nextcloud.deathstar.rocks/
```

Comment:
Why not as part of the `_links`-section?

Reply:
moved to `_links` section